backend/btrixcloud/orgs.py

```diff
@@ -722,14 +750,269 @@ async def update_read_only(self, org: Organization, update: OrgReadOnlyUpdate):
+
+        # Pages
+        count = await self.pages_db.count_documents(oid_query)
+        cursor = self.pages_db.find(oid_query)
+        export_stream_generators.append(json_items_gen("pages", cursor, count))
+
+        # Collections
+        count = await self.colls_db.count_documents(oid_query)
+        cursor = self.colls_db.find(oid_query)
+        export_stream_generators.append(
+            json_items_gen("collections", cursor, count, True)
+        )
+
+        export_stream_generators.append(json_closing_gen())
+
+        return StreamingResponse(stream.chain(*export_stream_generators))
+
+    async def import_org(
+        self,
+        stream_file_object,
+        ignore_version: bool = False,
+        storage_name: Optional[str] = None,
+    ):
+        """Import org from exported org JSON
+
+        :param stream: Stream of org JSON export
+        :param ignore_version: Ignore db version mismatch between JSON and db
+        :param storage_name: Update storage refs to use new name if provided
+        """
+        # pylint: disable=too-many-branches, too-many-statements
+
+        org_stream = json_stream.load(stream_file_object)
+        org_data = org_stream["data"]
+
+        # dbVersion
+        version_res = await self.version_db.find_one()
+        version = version_res["version"]
+        stream_db_version = org_data.get("dbVersion")
+        if version != stream_db_version and not ignore_version:
+            print(
+                f"Export db version: {stream_db_version} doesn't match db: {version}, quitting",
+                flush=True,
+            )
+            raise HTTPException(status_code=400, detail="db_version_mismatch")
+
+        # org
+        stream_org = org_data["org"]
+        stream_org = json_stream.to_standard_types(stream_org)
+        oid = UUID(stream_org["_id"])
+
+        existing_org: Optional[Organization] = None
+        try:
+            existing_org = await self.get_org_by_id(oid)
+        except HTTPException:
+            pass
+
+        if existing_org:
+            print(f"Org {oid} already exists, quitting", flush=True)
+            raise HTTPException(status_code=400, detail="org_already_exists")
+
+        new_storage_ref = None
+        if storage_name:
+            new_storage_ref = StorageRef(name=storage_name, custom=False)
+
+        org = Organization.from_dict(stream_org)
+        if storage_name and new_storage_ref:
+            org.storage = new_storage_ref
+        await self.orgs.insert_one(org.to_dict())
+
+        # Track old->new userids so that we can update as necessary in db docs
+        user_id_map = {}
+
+        # Users are imported with a random password and will need to go through
+        # the reset password workflow using their email address after import.
+        for user in stream_org.get("userDetails", []):  # type: ignore
+            try:
+                user_res = await self.user_manager.create_non_super_user(
```

Comment:
This method is removed in https://github.com/webrecorder/browsertrix/pull/1902, will need to be updated either in this PR or that one depending on what gets merged first

Reply:
Tested merging #1902 branch into this, shouldn't be too bad!